### PR TITLE
Clarified prune documentation regarding tags and interpolate

### PR DESCRIPTION
**Pull request by 3ygun**
In response to #13 
- [x] Clarified that tags are affected by whitelist and blacklist
- [x] Clarified what interpolate does by displaying the common logstash dynamic value syntax
  - [x] The `%{some_field}` should be in the `code` style < Don't know how to do :/
- [x] Clarified how to specify an exact field or value name

### CODE REVIEW

File: lib/logstash/filters/prune.rb [hunk 1]
```diff
@@ -16,11 +16,11 @@ class LogStash::Filters::Prune < LogStash::Filters::Base
   config_name "prune"
 
   # Trigger whether configuration fields and values should be interpolated for
-  # dynamic values.
+  # dynamic values (when resolving %{some_field}).
```
Comment (magnusbaeck) [inline lib/logstash/filters/prune.rb]: In Asciidoc you can use backticks (`) to format text as code (which I agree would be appropriate in this case).

File: lib/logstash/filters/prune.rb [hunk 2]
```diff
@@ -16,11 +16,11 @@ class LogStash::Filters::Prune < LogStash::Filters::Base
...
   # Probably adds some performance overhead. Defaults to false.
   config :interpolate, :validate => :boolean, :default => false
 
-  # Include only fields only if their names match specified regexps, default to empty list which means include everything.
+  # Include only fields and tags only if their names match specified regexps, default to empty list which means include everything.
```
Comment (magnusbaeck) [inline lib/logstash/filters/prune.rb]: The way you've reworded this sentence (and the corresponding one on line 32) could give the impression that the pruning acts on tag values themselves rather than the `tag` field, i.e. that you could say `whitelist_names => ["foo"]` to keep the "foo" tag but remove all other tags.

Would it be a good idea to reword this to avoid that ambiguity? Maybe add a sentence (here or elsewhere) similar to "The `tags` field that stores the event's tags is also subject to pruning."

**Comment by magnusbaeck** [comment]
If you don't mind terribly, could you amend the commit and update the commit message to not just say "Update prune.rb"? The PR's title ("Clarified prune documentation regarding tags and interpolate") would be much better.

**Comment by 3ygun** [comment]
Requested changes made.

**Comment by magnusbaeck** [comment]
Thanks! I'll look into making a 2.0.5 release with this PR.

**Comment by magnusbaeck** [comment]
> I'll look into making a 2.0.5 release with this PR.

2.0.5 apparently already exists (but hadn't been tagged), so it'll be 2.0.6.
